processing/src/main/java/io/druid/query/QueryInterruptedException.java

```diff
@@ -118,6 +119,8 @@ private static String getErrorCodeFromThrowable(Throwable e)
       return QUERY_CANCELLED;
     } else if (e instanceof TimeoutException) {
       return QUERY_TIMEOUT;
+    } else if (e instanceof ResourceLimitExceededException) {
```

Comment:
If we end up having a ton of our own exceptions, we could do something like `if instanceof DruidQueryException { return ((DruidQueryException) e).getStatusCode; }`

But I thought for now this wasn't needed